packages/client-s3-node/package.json

```diff
@@ -57,12 +58,12 @@
     "tslib": "^1.8.0"
   },
   "devDependencies": {
-    "@aws-sdk/client-documentation-generator": "^0.1.0-preview.2",
+    "@aws-sdk/client-documentation-generator": "^0.1.0-preview.1",
     "@types/jest": "^24.0.12",
-    "@types/node": "^10.0.0",
```

Comment:
Looks like your workspace isn't synced from upstream as this was fixed in https://github.com/aws/aws-sdk-js-v3/pull/301
Can you sync your workspace and regenerate S3 client?

Reply:
Nice catch